requirements/python.txt

```diff
@@ -3,17 +3,17 @@ future==0.18.3
 ipywidgets==8.0.4
 jupyterlab==3.6.1
 matplotlib==3.7.1
-mlflow[extras]==2.2.2
+mlflow[extras]==2.17.2
 opencv-python-headless==4.7.0.72
```

Comment:
curious, did you try to bump opencv-python-headless and wandb? or as a separate PR?